ui/src/reducers/index.js

```diff
@@ -150,6 +152,40 @@ export default (state = initialState, action) => {
         ...state,
         isUserDropdownOpen: !isUserDropdownOpen
       };
+    case UPDATE_DISABLED_APP:
+      const id = action.payload.id;
+      const disabled = action.payload.disabled;
```

Comment:
Could we use `isDisabled` rather than `disabled`? This is a boolean value right?

Reply:
done